### DISCUSSION TITLE
对于翻译中语句不通顺但无实际错误的地方是否可以或者说需要提交 PR 修改

### DISCUSSION THREAD

**Discussion by ryomahan** [reactjs/zh-hans.react.dev#1388] [Q&A]
问题如题，举例：

在 [这个链接](https://zh-hans.react.dev/learn/reusing-logic-with-custom-hooks#should-all-functions-called-during-rendering-start-with-the-use-prefix) 指向的「深入探讨」的开头有这么一句话：

> 如果函数没有调用任何 Hook，请避免使用 `use` 前缀。 而是 不带 `use` 前缀把它当成常规函数去写。例如下面的 `useSorted` 没有调用 Hook，所以叫它 `getSorted`：

原文是：

> If your function doesn’t call any Hooks, avoid the `use` prefix. Instead, write it as a regular function without the `use` prefix. For example, `useSorted` below doesn’t call Hooks, so call it `getSorted` instead:

从逐字翻译的角度来看没啥问题，但是怎么读都觉得不通顺，仅我个人的角度来看可以翻译成：

如果你创建的函数没有调用任何 Hook 方法，应避免使用 `use` 前缀，而是把它当成一个常规函数命名。例如下面的 `useSorted` 函数就没有调用任何 Hook 方法，所以我们可以使用 `getSorted` 来命名它：

我的翻译可能并不是最好的选择，但在应该更容易让人理解，文档中类似的地方还挺多的（我甚至一度怀疑是机翻），只是恰好读到这里所以就以此举例了。

所以我就想问，如果我读到了感觉不通顺的地方是否可以直接提 PR 进行修正还是说需要像翻译任务一样先提交 issue 在提交 PR。

@awxiaoxian2020 @Yucohny 因为之前两次 PR 分别是两位审核的所以就艾特一下两位了。

**Comment by Yucohny** [comment]
如果认为遇到了不通顺之处，直接提交 PR 即可，我会来 review。

**Comment by Huxpro** [comment]
当然欢迎！
